maintainers/maintainer-list.nix

```diff
@@ -12740,6 +12740,13 @@
     githubId = 1386642;
     name = "Noah Brenowitz";
   };
+  nbsp = {
+    email = "aoife@enby.space";
+    matrix = "@nbsp:enby.space";
+    github = "enbyspace";
+    githubId = 57151943;
+    name = "aoife cassidy";
+  };
```

Comment:
I would appreciate some clarification on why the original maintainer was removed. Have you considered adding yourself as an additional maintainer or including @sfr as an extra maintainer? It seems like @sfr only maintains `mus`, but could potentially be helpful in a supporting role. Has @sfr confirmed their willingness to be dropped from the team?

Reply:
@otavio Please note githubId has not changed: https://caius.github.io/github_id/#enbyspace. I can also personally attest this to be a simple name change as claimed.

Reply:
Thank you for providing me with the information. I apologize for any inconvenience.